main.go

```diff
@@ -34,5 +35,8 @@ func buildVersion(version, commit, date, builtBy string) string {
 	if builtBy != "" {
 		result = fmt.Sprintf("%s\nbuilt by: %s", result, builtBy)
 	}
+	if info, ok := debug.ReadBuildInfo(); ok && info.Main.Sum != "" {
+		result = fmt.Sprintf("%s\nmodule version: %s, checksum: %s", result, info.Main.Version, info.Main.Sum)
```

Comment:
Where does this information get used? Because this will be the build info for `goreleaser` itself, right?

Reply:
yeah, this is for `goreleaser --version`...